### PR TITLE
feat: add ability to hide loop icon

### PR DESCRIPTION
**Pull request by acherkashin**
In my project I need just play single sound and actually I don't need loop icon. SO I have added option to hide this icon.

### CODE REVIEW

File: src/components/AudioPlayer/AudioPlayer.tsx [hunk 1]
```diff
@@ -116,69 +119,69 @@ class AudioPlayer extends React.Component<AudioPlayerProps> {
...
-            <Grid md={2} item>
-                <Typography
-                    className={css(classes['player-text'], text)}
-                    align="center"
-                    noWrap
-                >
-                    {getFormattedTime(duration)}
-                </Typography>
-            </Grid>
+          {showLoopIcon && <Grid md={1} item>
```
Comment (JonatanSalas) [inline src/components/AudioPlayer/AudioPlayer.tsx:122]: Hi! Would you mind to wrap the `Grid` with parenthesis?

**Review by JonatanSalas — CHANGES_REQUESTED** [review]

**Comment by JonatanSalas** [comment]
@acherkashin thanks for your PR! Really glad to receive a contribution. As soon as you implement the requested change I will merge this into the core, and release a new `minor` version.

File: src/components/AudioPlayer/AudioPlayer.tsx (hunk 1 above)
Comment (acherkashin) [reply src/components/AudioPlayer/AudioPlayer.tsx:122]: @JonatanSalas Hi! To wrap like `{showLoopIcon  && (<Grid>...</Grid>)}`?

Comment (JonatanSalas) [reply src/components/AudioPlayer/AudioPlayer.tsx:122]: Yep! Anyway, there's a `.prettierrc.js` file that should handle this case 😅. Do you use `vscode` with `prettier` extension?

Comment (JonatanSalas) [reply src/components/AudioPlayer/AudioPlayer.tsx:122]: Let's resolve this, and I will review it in my local machine.

Comment (acherkashin) [reply src/components/AudioPlayer/AudioPlayer.tsx:122]: @JonatanSalas Yeah, I use `vscode` but don't use `prettier` extension. I will install it. Thank you!

**Comment by JonatanSalas** [comment]
New release, update to `2.1.0`: 

```bash
npm i @blackbox-vision/mui-audio-player@2.1.0
```

Once again, thank you for your contribution!

**Pull request merged**